Gemfile

```diff
@@ -56,6 +56,7 @@ group :development, :test do
   gem 'brakeman'
   gem 'bundler-audit'
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
```

Comment:
What do you need it for?

Reply:
I need this gem to execute:
`expect(response).to render_template(:index)`

Reply:
I see now, that I don't need this check, so will delete gem.